### ISSUE TITLE
Request: Add style option for empty workspaces

### ISSUE THREAD

**Issue by Mishio595** [Alexays/Waybar#377]
I would be really nice to have the option to style workspaces that are displayed via persistant_workspaces (i.e. that have no clients and are not active) independently. Perhaps something like `#workspaces button.empty` or something.

**Comment by Alexays** [comment]
Done in c2e9ed60913604a1b8f9b6f73d930cf8e002da47
You can use the `.persistant` class

**Comment by Mishio595** [comment]
I must be doing something wrong with this. I just updated
```
$ waybar -v
Waybar v0.6.8-8-g564fdcb (Jun 11 2019, branch 'master')
```

In my `$HOME/.config/waybar/style.css` I have
```css
#workspaces button {
    padding: 0 5px;
    color: #f8f8f2;
}

#workspaces button.persistant {
    color: #555;
}
```

The colour in the 2nd block isn't applied to empty workspaces that are displayed via persistant_workspaces

**Comment by Alexays** [comment]
Fixed with 028b184f7b68d4263e2ee789b5b6555ca60f24b1

**Comment by bklaase** [comment]
Great feature! Would it also be possible to have a format icon for it?
